### PR TITLE
created the landing page

### LINKED ISSUE #42: FE: Landing Page

### PR DESCRIPTION
**Pull request by panwaramita**
### What this PR does (required):
- Create the landing page user interface.
- issue : close #42 

### Screenshots / Videos (front-end only):
landingPage

### Any information needed to test this feature (required):
- Clone the repository.
- Ro to the client folder using cd client
- Run npm install
- The localhost:3000 page will open that contains the landing page instead of the login or signup page.

### Any issues with the current functionality (optional):
- When the user clicks on the find my dog sitter button nothing will be displayed. The reason is the changes are done only on the front end.

### CODE REVIEW

File: client/src/components/Navbar/useStyles.ts
Comment (ezeilo-su) [inline client/src/components/Navbar/useStyles.ts:8]: Let's remove this unused code

File: client/src/components/Navbar/useStyles.ts
Comment (ezeilo-su) [inline client/src/components/Navbar/useStyles.ts:17]: Let's clean up by remove unused code

File: client/src/pages/Home/Home.tsx
Comment (ezeilo-su) [inline client/src/pages/Home/Home.tsx:104]: Please add the `alt` attribute

**Review by ezeilo-su — COMMENTED** [review]
Looks good!

File: client/src/components/Navbar/useStyles.ts
Comment (panwaramita) [reply client/src/components/Navbar/useStyles.ts:8]: Done the changes

File: client/src/components/Navbar/useStyles.ts
Comment (panwaramita) [reply client/src/components/Navbar/useStyles.ts:17]: Done the changes

File: client/src/pages/Home/Home.tsx
Comment (panwaramita) [reply client/src/pages/Home/Home.tsx:104]: Done the changes

**Review by ezeilo-su — COMMENTED** [review]
Thank you for making the requested changes. Please wait for approval before merging.

File: client/src/pages/Home/Home.tsx
Comment (rajivtitus) [inline client/src/pages/Home/Home.tsx:76]: Please avoid the usage of CAPITAL CASE text for accessibility reasons 

Ditto where applicable

File: client/src/pages/Home/useStyles.ts
Comment (rajivtitus) [inline client/src/pages/Home/useStyles.ts:15]: 👍🏽

File: client/src/pages/Home/useStyles.ts
Comment (rajivtitus) [inline client/src/pages/Home/useStyles.ts:20]: I think it would be better to use relative units for responsiveness

File: client/src/pages/Home/Home.tsx
Comment (rajivtitus) [inline client/src/pages/Home/Home.tsx:78]: We can move the `LocalizationProvider` to a parent component so that it can be accessed across the entire app

File: client/src/pages/Home/Home.tsx
Comment (rajivtitus) [inline client/src/pages/Home/Home.tsx:69]: Please avoid the usage of inline styles

Ditto where applicable

File: client/src/helpers/APICalls/dropDates.ts
Comment (rajivtitus) [inline client/src/helpers/APICalls/dropDates.ts:11]:
```suggestion
  return await fetch(`/get-info`, fetchOptions)
```
We use hyphenated case for route names in JS :)

**Review by rajivtitus — COMMENTED** [review]
UI looks fantastic! Just a few minor comments for you to have a look 👍🏽

File: client/src/helpers/APICalls/dropDates.ts
Comment (panwaramita) [reply client/src/helpers/APICalls/dropDates.ts]: Done the changes.

File: client/src/pages/Home/Home.tsx
Comment (panwaramita) [reply client/src/pages/Home/Home.tsx]: move styling to useStyle page

File: client/src/pages/Home/Home.tsx
Comment (panwaramita) [reply client/src/pages/Home/Home.tsx]: Done the changes.

File: client/src/pages/Home/useStyles.ts
Comment (panwaramita) [reply client/src/pages/Home/useStyles.ts]: Done the changes.

File: client/src/pages/Home/Home.tsx
Comment (panwaramita) [reply client/src/pages/Home/Home.tsx]: Done the changes.

**Review by rajivtitus — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**
